RELEASE.md

```diff
@@ -1,6 +1,7 @@
 # Upcoming Release
 
 ## Major features and improvements
+* Dropped Python 3.8 support.
```

Comment:
Don't forget to add your name to the contributor list as well! 😄

Reply:
Yay why not 🚀